app/controllers/admin/topics_controller.rb

```diff
@@ -312,6 +312,34 @@ def assign_team
     end
   end
 
+  def split_topic
+    parent_topic = Topic.find(params[:topic_id])
+    parent_post = Post.find(params[:post_id])
+
+    topic = parent_topic.forum.topics.new(
+      name: "Split from #{parent_topic.name}",
```

Comment:
One suggestion would be to use an `i18n` key here so this message can be customized for each locale.